BioForm/profiles/views.py

```diff
@@ -73,11 +71,9 @@ def create_record(request):
 
     if request.method == "POST":
         form = BioForm(request.POST)
-        #form.cleaned_data["username"] = request.user.username
```

Comment:
This was a good approach to make sure a user can only update his/her own bio. Add this functionality again.

Reply:
form.cleaned_data is only accessible after form has been verified so I had to remove this line as it threw an error and form wouldn't be verified without a username.

Instead I took care of this feature in line 80 by "form.fields["user"].initial = request.user.id"

Reply:
You can change it after validation of the form, on line 80 you are setting the initial value and that will be returned back from the frontend. On frontend, a user easily can change it so it's not that much effective.